portfolio/src/main/webapp/tmp.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Tmp</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,700;0,900;1,100;1,200;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div>
+          <ul class="nav-links">
+            <li><a class="active"  href="tmp.html">Tmp</a></li>
+            <li><a href="chat.html">Chat</a></li>
+            <li><a href="contacts.html">Contact</a></li>
+            <li><a href="index.html">About</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <div class="content">
+      <p>Click here to get a random message:</p>
+      <div id="greeting-container"></div>
+      <button onclick="addRandomGreeting()">Hello</button>
+      <br>
+      <p>Click here to fetch a data:</p>
```

Comment:
"data" is plural, so "fetch data". Better: describe what kind of data will be fetched.